.circleci/config.yml

```diff
@@ -8,3 +8,23 @@ jobs:
       - run:         
           name: Run Tests
           command:  docker-compose run web pytest --cov-config=.coveragerc --cov=ops_site --cov-report term-missing
+      - run:
+          name: Install cf cli
+          command: |
+            mkdir -p $HOME/bin
+            export PATH=$HOME/bin:$PATH
+            curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary" | tar xzv -C $HOME/bin
+            cf install-plugin autopilot -f -r CF-Community
+      - run:
+          name: Deploy to cloud.gov
+          command: |
+            if [[ -n "${CIRCLE_BRANCH}" && "${CIRCLE_BRANCH}" != "master" ]]; then
+              export PATH=$HOME/bin:$PATH
+              cf login -a https://api.fr.cloud.gov -u $DEV_USER -p $DEV_PASSWORD -o $ORG_NAME -s $SPACE_NAME
+              echo "Login successful."
+              cf zero-downtime-push opre-ops-test -f manifest.yml
+              echo "Deploy successful."
+            else
+              echo "Skipped"
+            fi
```

Comment:
[question]: So what we are doing here is that we are deploying to this cloud environment when something is being push to some branch.  This will be triggered?

Reply:
Ah good catch, I had it set up this way to test, but we should make it so on a merge to a new `development` branch this gets deployed. Does that sound good to you?

Reply:
I am looking back on what we agreed on as our deployment process.  In [continuous deployment to dev](https://github.com/HHS/OPRE-OPS/blob/main/docs/how_we_work/deploy_flow.md#continuous-deployment-to-dev):
>Our plan is to allow any dev to trigger a deploy to the dev app via a git tag. (See "Executing workflows for a git tag" in the Circle docs.) This will give devs the flexibility to deploy to dev and test out new code at any time, without waiting for a review process or a git merge process.

So it seems like we are choosing any git-tag to deploy to.  If we want to alter from this, we probably need to update this piece of documentation.

Reply:
🤔 oh man, I don't remember this at all lol. Many things have left my brain over the last couple of months.